common-ui/src/main/java/com/duckduckgo/mobile/android/themepreview/ui/component/ComponentViewHolder.kt

```diff
@@ -99,6 +99,16 @@ sealed class ComponentViewHolder(val view: View) : RecyclerView.ViewHolder(view)
                 setLeadingIconClickListener { Snackbar.make(view, "Leading Icon clicked", Snackbar.LENGTH_SHORT).show() }
                 setOnCheckedChangeListener { view, isChecked -> Snackbar.make(view, "Switch checked: $isChecked", Snackbar.LENGTH_SHORT).show() }
             }
+
+            view.findViewById<OneLineListItem>(R.id.oneLineListItemDisabled).apply {
+                setClickListener { Snackbar.make(this.rootView, component.name, Snackbar.LENGTH_SHORT).show() }
+                isEnabled = false
+            }
+
+            view.findViewById<OneLineListItem>(R.id.oneLineListItemCustomTextColor).apply {
+                setClickListener { Snackbar.make(this.rootView, component.name, Snackbar.LENGTH_SHORT).show() }
+            }
```

Comment:
Why do we need this now?

Reply:
These two are variants that weren't showcased earlier:
- DIsabled item (so text color is different to the enabled one)
- Item with custom text color